tools/refseq_masher/matches.xml

```diff
@@ -0,0 +1,293 @@
+#set $_reverse_ext = '.fastq.gz' if $re.match(r'.*\.gz$', $input.reverse.name) else '.fastq'
+#set $_reverse = '"{}_2{}"'.format($re.sub(r'_[12]\..+$', '', $input.reverse.name), $_reverse_ext)
+#set $input_files = '{} {}'.format($_forward, $_reverse)
+  ln -s "$input.forward" $_forward &&
+  ln -s "$input.reverse" $_reverse &&
+#elif $input.type == 'single'
+#set $input_files = '"{}"'.format($input.single.name)
+  ln -s "$input.single" $input_files &&
+#elif $input.type == 'paired_collection'
+#set $_forward = '"{}_1.{}"'.format($input.paired_collection.name, $input.paired_collection.forward.ext)
+#set $_reverse = '"{}_2.{}"'.format($input.paired_collection.name, $input.paired_collection.reverse.ext)
+#set $input_files = '{} {}'.format($_forward, $_reverse)
+  ln -s "$input.paired_collection.forward" $_forward &&
+  ln -s "$input.paired_collection.reverse" $_reverse &&
+#end if
+
+refseq_masher 
+  $adv.verbosity 
+  matches 
+  --output refseq_masher-matches.${adv.output_type}
+  --output-type $adv.output_type
+  --top-n-results $top_n_results
+#if $adv.min_kmer_threshold
+  --min-kmer-threshold $adv.min_kmer_threshold
+#end if
+  -T "\${TMPDIR:-/tmp}"
+  $input_files
+]]>
+  </command>
+  <inputs>
+    <conditional name="input">
+      <param name="type" type="select" label="Sequence input type">
+        <option value="fasta">Genome FASTA</option>
+        <option value="paired">Paired-end FASTQs</option>
+        <option value="single">Single-end FASTQ</option>
+        <option value="paired_collection">Paired-end FASTQ collection</option>
+      </param>
+      <when value="fasta">
+        <param name="fasta"
+          type="data" format="fasta"
+          optional="false"
+          label="Genome FASTA file"
+          />
+      </when>
+      <when value="paired">
+        <param name="forward"
+          type="data" format="fastq,fastqsanger,fastqillumina,fastqsolexa"
+          optional="false"
+          label="Forward FASTQ file"
+          />
+        <param name="reverse"
+          type="data" format="fastq,fastqsanger,fastqillumina,fastqsolexa"
+          optional="false"
+          label="Reverse FASTQ file"
+          help="File format must match the Forward FASTQ file"
+          />
+      </when>
+      <when value="single">
+        <param name="single"
+          type="data" format="fastq,fastqsanger,fastqillumina,fastqsolexa"
+          optional="false"
+          label="Single-end FASTQ file"
+          />
+      </when>
+      <when value="paired_collection">
+        <param name="paired_collection"
+          type="data_collection" format="txt,fastq,fastqsanger,fastqillumina,fastqsolexa"
+          collection_type="paired"
+          optional="false"
+          label="Paired-end FASTQ collection"
+          />
+      </when>
+    </conditional>
+    <param name="top_n_results"
+      type="integer"
+      min="0"
+      value="5"
```

Comment:
Could the default be increased (maybe 20)?

Reply:
Changed in f3cd512

Reply:
Thanks.